front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -104,48 +104,48 @@ const IdeasNewPageWithJSONForm = () => {
     }
 
     location_point_geojson = await getLocationGeojson(initialFormData, data);
-    if (!isNilOrError(authUser)) {
-      addIdea(
-        {
-          ...data,
-          location_point_geojson,
-          project_id: project?.id,
-          publication_status: 'published',
-          phase_ids:
-            phaseId && isModerator({ data: authUser }) ? [phaseId] : null,
-        },
-        {
-          onSuccess: (idea) => {
-            const ideaId = idea.data.id;
-
-            // Check ParticipationMethodConfig for form submission action
-            if (
-              project?.attributes.process_type === 'timeline' &&
-              !isNilOrError(phases)
-            ) {
-              // Check if URL contains specific phase_id
-              const phaseUsed =
-                phases.find((phase) => phase.id === phaseId) ||
-                getCurrentPhase(phases);
-              if (!isNilOrError(phaseUsed)) {
-                getMethodConfig(
-                  phaseUsed?.attributes?.participation_method
-                ).onFormSubmission({
-                  project,
-                  ideaId,
-                  idea,
-                  phaseId: phaseUsed.id,
-                });
-              }
-            } else if (!isNilOrError(project)) {
+    addIdea(
+      {
+        ...data,
+        location_point_geojson,
+        project_id: project?.id,
+        publication_status: 'published',
+        phase_ids:
+          phaseId && !isNilOrError(authUser) && isModerator({ data: authUser })
+            ? [phaseId]
```

Comment:
This is the change I made here - I removed the "!isNilOrError(authUser) wrapper around submission code, and added it here where it's used instead.